src/config/kernel.md

```diff
@@ -36,6 +36,14 @@ pre-installed on every Void Linux system. This utility runs the necessary
 [hooks](#kernel-hooks) when removing old kernels. Note that `vkpurge` does not
 remove kernel *packages*, only particular *kernels*.
 
+## Using a specific kernel series
+
+To use a kernel series other than the current default, use
```

Comment:
"and be able to remove the default" ? Because this isn't necessary if you only want to install another series. It should be made clear that people can have multiple kernel series installed, because the "cleaning up my system" scenario isn't the important one here.

Reply:
Point taken. "To change the default kernel series, and be able to remove other kernel series, ..."?

Reply:
"To be able to remove the default kernel series, after installing a different one, it is necessary to change the default" perhaps makes it clearer that this is only necessary if you want to remove stuff? I'm thinking of newbies who see this and think they need to remove the default kernel for some reason.

Reply:
Thanks, PR updated (with changed subsection heading).